apiserver/facades/client/bundle/bundle_test.go

```diff
@@ -1868,7 +1889,41 @@ relations:
 - - wordpress:db
   - mysql:mysql
 `[1:]
-	expectedResult := params.StringResult{nil, output}
+	expectedResult := params.StringResult{Result: output}
+
+	c.Assert(result, gc.Equals, expectedResult)
+	s.st.CheckCall(c, 0, "ExportPartial", s.st.GetExportConfig())
+}
+
+func (s *bundleSuite) TestExportCharmhubBundle(c *gc.C) {
+	model := s.newModel("iaas", "ch:wordpress", "ch:mysql")
+	model.SetStatus(description.StatusArgs{Value: "available"})
+
+	result, err := s.facade.ExportBundle()
+	c.Assert(err, jc.ErrorIsNil)
+
+	output := `
+series: xenial
+applications:
+  mysql:
+    charm: mysql
+    num_units: 1
```

Comment:
Shouldn't we have a channel here? Or is it empty to indicate "use whatever"? If we do know the channel (even if it's the default one) shouldn't we always include it so that the bundle can be re-deployed as-is (with the same channels) even if the charm authors monkey around with channels between deployments?

Reply:
I'll fix, it's essentially it's the test code.